signup.html

```diff
@@ -35,62 +27,77 @@ <h1>Signup Form</h1>
           <input type="password" id="password" name="password" />
           <span class="error" id="passwordError"></span><br />
           <div class="clearfix">
-            <button type="submit" class="signupbtn">Submit</button>
+            <button type="button" class="signupbtn" onclick="validateForm()">
+              Submit
+            </button>
           </div>
         </div>
       </form>
-      <script>
-        document.getElementById("myForm").onsubmit = function (event) {
-          let isValid = true;
+      <div class="success-message" id="successMessage"></div>
+    </section>
+    <footer>
+      <div class="footer">
+        <a href="https://www.linkedin.com/in/fazila-abbasi-a07b18264/"
+          >Link to My LinkedIn Profile!</a
+        >
+      </div>
+    </footer>
+
+    <script>
+      function validateForm() {
+        let isValid = true;
+
+        // Reset error messages and field styles
+        document.getElementById("nameError").textContent = "";
```

Comment:
Store all the **document.getElementById("...")** in their respective variables and use those variables for retrieving the attribute.

Reply:
fix